code/processes/tailer.q

```diff
@@ -46,9 +47,48 @@ getprocs:{[x;y]
         }
 
 .servers.register[.servers.procstab;.tailer.tailreadertypes;1b]
+.servers.register[.servers.procstab;.tailer.tailsorttypes;1b]
 
 \d .
 
+endofdaysort:{[pt]
+  /- function to be called when no tailsort process can be connected, to carry out eod sort from tailer
+  .lg.o[`eodsort;"attempting to complete eod sort from tailer"];
+  /- get tailsort code
+  .proc.loadf [getenv[`KDBCODE],"/processes/tailsort.q"];
+  /- sleep allows for multiple tailers to save to HDB at different times uninterrupted
+  .os.sleep[`$first .proc.params[`segid]];
```

Comment:
I don't think this will be very robust. Couple of issues here - I think we've decided that segment IDs may not be numerical, so this probably wouldn't work at all in that case.

Additionally, I think this is only going to stagger tailers by 1 second? what if it takes e.g. 10 seconds for a tailer to write to the hdb? or 5 mins etc.

I think for this to be robust there would need to be some mechanism to ensure that other tailers have finished their writes, rather than just waiting a bit and hoping